source/_components/media_player.firetv.markdown

```diff
@@ -26,26 +26,23 @@ Steps to configure your Amazon Fire TV stick with Home Assistant:
   - From the main (Launcher) screen, select Settings.
   - Select System > About > Network.
 
-To add FireTV to your installation, Note your device name, and add the following to your `configuration.yaml` file:
+To add Fire TV to your installation, add the following to your `configuration.yaml` file:
 
 ```yaml
 # Example configuration.yaml entry
 media_player:
-  # a device that does not require ADB authentication
+  # use the Python ADB implementation
   - platform: firetv
     name: Fire TV 1
     host: 192.168.0.111
+    adbkey: "/config/android/adbkey"
+    apps:
```

Comment:
For simplicity, we don't show the optional parameters in the configuration example.

Reply:
Should I change it back to having 3 example configurations? I removed the first configuration with only required parameters because all Fire TV devices require authentication now, so that configuration wouldn't work. Either `adbkey` (use Python ADB implementation) or `adb_server_ip` (use an ADB server) needs to be provided.

Reply:
Has the component been updated to reflect the fact that either `adbkey`/`adb_server_ip` is required? If it has not, we should make the schema change there.

Reply:
I suppose I made too strong of a statement above: I can't say for sure that every single Fire TV device out there requires authentication. It's possible that someone out there has blocked their Fire TV stick from accessing the internet and so it's running an old version of the OS, and requiring one of those two config entries would break their setup. 

What do you think? Either way, I can provide a basic configuration first and then one or two full configurations.

Reply:
In that case, I would say that we should then show all three ways of connecting (unauthenticated, plus the two different ways to authenticate) and then have a full example at the bottom with the `apps` section.

Reply:
Ok, I'll make those changes. 

In the meantime, could you please merge this pull request and I'll rebase this one on top of it: 
https://github.com/home-assistant/home-assistant.io/pull/8722

Reply:
Done.